cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/RemoteTaskExecutor.java

```diff
@@ -202,4 +203,12 @@ private ByteBuffer encodeTaskRequest(RunnableTaskRequest request) throws IOExcep
 
     throw new IllegalArgumentException("Unsupported content encoding " + encoding);
   }
+
+  private void setWorkerUrl(WorkerType workerType) {
```

Comment:
nit: Not really needed. More readable if the body is moved to constructor.

Reply:
Done